Core/GDCore/Project/LayoutEditorPreviewer.h

```diff
@@ -1,6 +1,12 @@
 #ifndef LAYOUTEDITORPREVIEWER_H
 #define LAYOUTEDITORPREVIEWER_H
+
 #if defined(GD_IDE_ONLY) && !defined(GD_NO_WX_GUI)
+
+#ifdef __WINDOWS__

```

Comment:
Can you try with just `#ifdef WINDOWS`? `WINDOWS` macro should be defined during compilation by GD.